src/main/java/no/sikt/nva/pubchannels/handler/CreateJournalHandler.java

```diff
@@ -17,6 +19,7 @@
 import java.net.http.HttpClient;
 import java.util.Map;
 
+import static nva.commons.core.attempt.Try.attempt;
 import static nva.commons.core.paths.UriWrapper.HTTPS;
 
 public class CreateJournalHandler extends ApiGatewayHandler<CreateJournalRequest, Void> {
```

Comment:
I see that both handlers now have pretty similar methods for generating uri's. Should we consider a utility class for generating uri's?